bokeh/client/session.py

```diff
@@ -153,11 +153,16 @@ def __init__(self, session_id=DEFAULT_SESSION_ID, io_loop=None, url=DEFAULT_SERV
         self._connection = ClientConnection(session=self, io_loop=io_loop, url=url)
 
         self._current_patch = None
+        self._callbacks = []
 
     def _attach_document(self, document):
         self._document = document
         self._document.on_change(self._document_changed)
 
+    def add_periodic_callback(self, cb, callback_time):
```

Comment:
@havocp  Sounds good. Having it elsewhere from session is actually better.. I'll play around with your suggestions. What I'm not terribly keen on is having to use `document` explicitly.. (more or less like having to use document.add(..) is not optimal too). Maybe we can have also a `add_periodiodic_callback` (or perhaps a `register_periodic_callback`) decorator? That you can:

```
@add_periodiodic_callback(document)
def my_cb():
    do_my_stuff_here
```

that would use `curdoc()` is document is not explicitly passed.